core/src/components/checkbox/checkbox.ionic.vars.scss

```diff
@@ -1,2 +1,20 @@
+@import "../../themes/ionic.globals.ionic";
+
 // Ionic Checkbox Variables
-// --------------------------------------------------
\ No newline at end of file
+// --------------------------------------------------
+
+/// @prop - The default width and height of the checkbox
+$checkbox-ionic-size: 24px !default;
+
+/// @prop - The background color of the checkbox icon when the checkbox is unchecked
+$checkbox-ionic-icon-background-color-off: $background-color-step-450 !default;
```

Comment:
This border color is based from the Figma design color of `#9A9A9A`. 

Note: OS can customize the color palette to actually apply their desired color.

`$background-color-step-450` evaluates to `#8C8C8C` and provides a good visual default to an existing Ionic Framework variable.

Reply:
any reason for this over 	`--ion-color-step-400: #999999;`?

Reply:
![diff](https://github.com/ionic-team/ionic-framework/assets/6577830/bf3ee2a9-f39c-43d8-8c47-488012aa23e4)

Reply:
I'm ok with updating to step-400 👍 I had looked at both and simply committed. Without the design tokens connected from Figma, we will have to pick what is reasonably close & adjust as that architecture exists.